packages/procdot.vm/procdot.vm.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>procdot.vm</id>
+    <version>1.22.57</version>
+    <authors>Christian Wojner</authors>
+    <description>Creates visual graphs from procmon output</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="7zip-15-05.vm" />
```

Comment:
This is an issue if you install [7z-nsis.vm](https://github.com/mandiant/VM-Packages/tree/main/packages/7z-nsis.vm) as they would conflict. Options:
- to download and use a portable 7zip.
- To ask procdot author if he would mind releasing a version without password.
- Make 7z-nsis the default:
  - It may be enough to change it in the flare-vm config and ensure we are using it everywhere. 
  - another better alternative is to create a 7zip package which has as dependency the concrete zip package and use this in the flare-vm config. This makes it much easier to switch

Reply:
I created a `7zip` library package with `7zip-nsis` as the default: https://github.com/mandiant/VM-Packages/pull/981

That will need to be merged in before this PR will work.